RUFAS/output_manager.py

```diff
@@ -487,6 +520,27 @@ def save_variables(self, save_path: str, dir_path: str,
             file_path = os.path.join(save_path, self._generate_file_name(f"saved_variables_{filter_file}", "json"))
             self._dict_to_file_json(filtered_pool, file_path)
 
+    def dump_variables_to_csv_files(self, path: str) -> None:
+        """
+        Dumps variables_pool into one csv file per variable in the given path to a directory.
+
+        Parameters
+        ----------
+        path : str
+            Path to the output directory for the OutputManager.
+
+        """
+        pool = self.variables_pool
+        variables_csv_path = Path(path) / "CSVs" / "om" / "variables"
+        try:
+            variables_csv_path.mkdir(parents=True, exist_ok=True)
+        except Exception as e:
+            raise e
+
+        for key, value in pool.items():
+            csv_file_path = os.path.join(variables_csv_path, self._generate_file_name(key, "csv"))
+            self._dict_to_file_csv({key: value}, csv_file_path)
```

Comment:
I think this is more efficient.
```suggestion
        for key in pool.keys():
            csv_file_path = os.path.join(variables_csv_path, self._generate_file_name(key, "csv"))
            self._dict_to_file_csv(pool[key], csv_file_path)
```

Reply:
done.